Source/Evolve/WarpXEvolve.cpp

```diff
@@ -495,7 +495,7 @@ void WarpX::HandleParticlesAtBoundaries (int step, amrex::Real cur_time, int num
     mypc->ContinuousFluxInjection(cur_time, dt[0]);
 
     mypc->ApplyBoundaryConditions();
-    m_particle_boundary_buffer->gatherParticlesFromDomainBoundaries(*mypc);
+    if (m_eb_enabled) { m_particle_boundary_buffer->gatherParticlesFromDomainBoundaries(*mypc); }
```

Comment:
This was not inside #ifdef AMREX_USE_EB previously. So does it make sense to add `if (m_eb_enabled)`?

Reply:
I think this only made sense if eb is enabled.

Reply:
@roelof-groenewald in `Python_particle_reflection`, does that test scrape particles from a (domain) boundary without using EBs?

Maybe I should remove this then here again...

Reply:
Indeed that was a bug